storage/pathmanager/pathManager_test.go

```diff
@@ -11,66 +12,82 @@ func TestNewPathManager_EmptyPruningPathTemplateShouldErr(t *testing.T) {
 	t.Parallel()
 
 	pm, err := pathmanager.NewPathManager("", "shard_[S]/[I]", "db")
-	assert.Nil(t, pm)
+	assert.True(t, check.IfNil(pm))
```

Comment:
please revert to assert.Nil to all errors
assert.True is not good for nil checks as it requires more time to figure out the issue looking at the tests logs.

Reply:
reworked all tests from `storage` package. Left `check.IfNil(pm)` only when the tested function returned an interface.